src/components/SearchAndFilter.js

```diff
@@ -47,11 +47,17 @@ class SearchAndFilter extends React.Component {
       <React.Fragment key={length}>
         <label>Ingredient: </label>
```

Comment:
wrap the label around the input so that clicking on the label will put the cursor into the input box

e.g. `<label> Ingredient: <input .... /> </label>`